### PR TITLE
Bug Fix Grab Bag

### PR DESCRIPTION
**Pull request by gerth2**
A number of random bug fixes:
1) Pulled in matt's latest libcam .so
2) Added some more guardrails to settings on the ov9281 right before the JNI layer to prevent footguns in the UI
3) Fixed up some NT & driver-mode or pipeline switching issues

### CODE REVIEW

File: photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java [hunk 1]
```diff
@@ -114,20 +117,41 @@ public double getFOV() {
...
-        lastExposureMode = cameraAutoExposure;
-        // TODO (Matt) -- call LibCameraJNI's auto exposure function, when that exists
-        LibCameraJNI.setAutoExposure(cameraAutoExposure);
+        lastAutoExposureActive = cameraAutoExposure;
+
+        // HACK!
+        // OV 9281 does not currently support autoexposure in our driver
+        // Just skip setting it silently
+        if (sensorModel == LibCameraJNI.SensorModel.OV9281) {
+            lastAutoExposureActive = false;
```
Comment (mcm001) [inline photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java:126]: Does it still not?

Comment (gerth2) [reply photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java:126]: Yea I'm still having issues with https://github.com/PhotonVision/photon-libcamera-gl-driver/suites/10144555465/artifacts/495489276

Comment (mcm001) [reply photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java]: @gerth2 we good now?

**Review by mcm001 — APPROVED** [review]

File: photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java (hunk 1 above)
Comment (gerth2) [reply photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java]: yup all good

**Comment by mcm001** [comment]
After a through vibe check (and a test with my Pi 3 + picam v1) I give this PR two thumbs up

**Review by mcm001 — APPROVED** [review]

**Pull request merged**